tests/hpc/slurm_slave.pm

```diff
@@ -28,7 +28,18 @@ sub run ($self) {
         zypper_ar(get_required_var('DEVEL_TOOLS_REPO'), no_gpg_check => 1);
         zypper_call('in iputils python');
     }
+    my %users = (
+        'user_1' => 'sebastian',
+        'user_2' => 'egbert',
+        'user_3' => 'christina',
+        'user_4' => 'jose',
+    );
 
+    ##Add users TODO: surely this should be abstracted
+    script_run("useradd $users{user_1}");
+    script_run("useradd $users{user_2}");
+    script_run("useradd $users{user_3}");
+    script_run("useradd $users{user_4}");
```

Comment:
Users should appear among all the nodes of the cluster and their uid/gid
should match. It seems that the uid/gid are assigned correctly.

This used to work without the creation of the users on compute nodes for
SLE15SP4 and below but apparently it shouldnt. This commit just makes things
right.

---

Where could I find docs describing this (the fact that each user shall be created on each compute node)?

Reply:
https://slurm.schedmd.com/accounting.html#infrastructure